src/command_modules/azure-cli-rdbms/setup.py

```diff
@@ -29,7 +29,7 @@
 
 DEPENDENCIES = [
     'azure-cli-core',
-    'azure-mgmt-rdbms==1.3.0',
+    'azure-mgmt-rdbms==1.4.0',
```

Comment:
You cannot use `azure-mgmt.-rdbms 1.4.0` due to a bug that will cause this to fail on Homebrew installation. Please bump to 1.4.1

Reply:
done